src/dto/odp/dto/common/contact_info.py

```diff
@@ -9,5 +9,5 @@ class ContactInfo(BaseModel):
     contact: str
     """Contact in the form `Firstname Lastname <email>`"""
 
-    organization: Optional[str] = None
+    organisation: Optional[str] = None
```

Comment:
Have you checked the usage of this in our code and catalog(db) ?

Reply:
Done